flake.nix

```diff
@@ -24,6 +24,7 @@
     });
     # TODO: disable bios-related tests on aarch64...
     checks = forAllSystems (system: let
+      nixpkgs = builtins.getFlake "nixpkgs/nixos-unstable";
```

Comment:
I think this is impure and not allowed. I think we need a second nixpkgs that is locked in a lockfile...